solvers/stereo_imaging/time_window_pruned_stereo_milp/src/products.py

```
@@ -0,0 +1,289 @@
+    tris: list[TriStereoSet] = []
+    summary = ProductSummary()
+    summary.approximation_flags = {
+        "overlap_method": "polar_grid_area_uniform",
+        "overlap_grid_angles": int(config.get("overlap_grid_angles", 8)),
+        "overlap_grid_radii": int(config.get("overlap_grid_radii", 3)),
+        "strip_sample_step_s": strip_step_s,
+        "pixel_scale_secant_correction": True,
+        "note": "Overlap is grid-approximated; pixel scale includes off-nadir secant correction.",
+    }
+
+    for (sat_id, target_id, interval_id), group in groups.items():
```

Ruff (0.15.11)

[warning] Loop control variable `interval_id` not used within loop body

Rename unused `interval_id` to `_interval_id`

(B007)